Maestro/subscriptions.json

```
@@ -607,6 +607,23 @@
       "action": "github-runtime-main-to-runtimelab-mirror",
       "actionArguments": {}
     },
+    // Automate merging runtime release/6.0-rc* branches back into release/6.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/runtime/blob/release/6.0-rc*/**/*"
```

```suggestion
        "https://github.com/dotnet/runtime/blob/release/6.0-rc/**/*"
```

This isn't actually a glob. it just lops off `/**/*` and does a `StartsWith`